examples/angular_bazel_architect/.editorconfig

```diff
@@ -0,0 +1,13 @@
+# Editor configuration, see https://editorconfig.org
```

Comment:
Don't have this in the other examples so remove it from here? Could use one in the root looks like.